bears/jinja2/Jinja2Bear.py

```diff
@@ -0,0 +1,271 @@
+import re
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result, RESULT_SEVERITY
+from coalib.results.Diff import Diff
+
+
+def generate_spacing_diff(file, filename, line, line_number,
+                          match_object, required_spacing):
+    '''
+    Generate a diff for incorrectly spaced control or variable tags.
+
+    :param match_object:     the match object generated by a match on the
+                             regular expression control_start_regex,
+                             control_end_regex or variable_regex in the
```

Comment:
Actually, rather than saying it's from those regex, better is to mention it's a Match object with the named groups: start, content, end as that's what matters here.
Otherwise, a user would have to go back, read the regex and see what all can be queried